sdk/core/System.ClientModel/README.md

```diff
@@ -0,0 +1,59 @@
+# System.ClientModel shared client library for .NET
+
+`System.ClientModel` provides shared primitives, abstractions, and helpers for .NET service client libraries.
+
+## Getting started
+
+Typically, you will not need to install `System.ClientModel`.
+it will be installed for you when you install one of the client libraries using it.
+
+### Install the package
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/).
```

Comment:
Seems the "Getting started" summary should go here.  Otherwise, we should us the same conventions as the other libraries, showing the CLI call.